src/main/java/com/minecolonies/coremod/client/gui/WindowHutGuardTower.java

```diff
@@ -9,255 +9,484 @@
 import com.minecolonies.blockout.views.SwitchView;
 import com.minecolonies.coremod.MineColonies;
 import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuards;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuards.GuardJob;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuards.GuardTask;
+import com.minecolonies.coremod.colony.buildings.views.MobEntryView;
+import com.minecolonies.coremod.network.messages.GuardRecalculateMessage;
 import com.minecolonies.coremod.network.messages.GuardScepterMessage;
 import com.minecolonies.coremod.network.messages.GuardTaskMessage;
+import com.minecolonies.coremod.network.messages.MobEntryChangeMessage;
 import net.minecraft.client.entity.EntityPlayerSP;
 import net.minecraft.util.math.BlockPos;
 import org.jetbrains.annotations.NotNull;
 
 import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.List;
 
-import static com.minecolonies.api.util.constant.TranslationConstants.COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_OFF;
-import static com.minecolonies.api.util.constant.TranslationConstants.COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_ON;
+import static com.minecolonies.api.util.constant.TranslationConstants.*;
 
 /**
- * Window for the guardTower hut.
+ * Our building hut view.
+ *
+ * @author Asherslab
  */
 public class WindowHutGuardTower extends AbstractWindowWorkerBuilding<AbstractBuildingGuards.View>
 {
-    /**
-     * Id of the list of the patrol points in the GUI.
-     */
-    private static final String LIST_LEVELS = "positions";
+    //// ---- GUI Constants ---- \\\\
+    //GUI Lists
+    private static final String  GUI_ELEMENT_LIST_LEVELS    = "positions";
+    private static final String  GUI_ELEMENT_LIST_MOBS      = "mobs";
+    //GUI List Elements
```

Comment:
Can you move those to the GUI constants class?

Reply:
Done